docs/contributors-to-ushahidi.md

```diff
@@ -2,9 +2,9 @@
 
 There are many ways to get involved with Ushahidi. You can read about how to contribute in [our docs](https://docs.ushahidi.com/platform-developer-documentation/contributing-or-getting-involved).
 
-This is a list of people who have contributed to Ushahidi. Not everyone in this list contributed code. Some folks here are designers, and they designed different components of the Ushahidi platform to make it better. Some did things that improved the organization and enabled others to thrive, like writing docs, blog posts, or perform business functions for our non profit. Many developed code and reviewed other people's code. A lot of people opened new bug tickets or provided ideas.
+This is a list of people who have contributed to Ushahidi. Not everyone in this list contributed in code. Some folks here are designers, and they designed different components of the Ushahidi platform to make it better. Some did things that improved the organization and enabled others to thrive, like writing docs, blog posts, or perform business functions for our non profit. Many developed codes and reviewed other people's code. A lot of people opened new bug tickets or provided ideas.
```

Comment:
keep code singular (it's a difference in writing style only but it's the one we tend to use)